packages/textfield/src/textfield.css

```diff
@@ -52,6 +52,8 @@ textarea {
     block-size: auto;
     word-break: break-word;
     opacity: 0;
+    white-space: pre-line;
+    overflow: clip;
```

Comment:
Is this required with the change in the `render` method? Less is always more 😉

Reply:
`white-space: pre-line;` is still requied, but yeah we can remove `overflow: clip;`